### PR TITLE
template builder - null handler

### PR DESCRIPTION
**Pull request by aksh-h**

### CODE REVIEW

File: src/AzureDevOpsRestApi/Viewmodel/Extractor/KeyConfig.cs [hunk 1]
```diff
@@ -1,11 +1,13 @@
-﻿using System.Collections.Generic;
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
 
 namespace AzureDevOpsAPI.Viewmodel.Extractor
 {
     public class KeyConfig
     {
-        public class Keys
+        public class keys
```
Comment (Damovisa) [inline src/AzureDevOpsRestApi/Viewmodel/Extractor/KeyConfig.cs:8]: Any reason this is no longer capitalized? I though it was good practice to capitalize class names.

**Review by Damovisa — CHANGES_REQUESTED** [review]
Let me know if I'm missing something with the `Keys` class. 😄

File: src/AzureDevOpsRestApi/Viewmodel/Extractor/KeyConfig.cs (hunk 1 above)
Comment (aksh-h) [reply src/AzureDevOpsRestApi/Viewmodel/Extractor/KeyConfig.cs:8]: Yes, you are right!!, It should have been capitalize 🔠  
There was a problem with deserializing the keys, so I tried to lowercase this.

Later I tried the ```JsonProperty``` to resolve that - as you can see below. 😄 I'll correct this.

**Comment by aksh-h** [comment]
> Let me know if I'm missing something with the `Keys` class. 😄

Thanks Damian, Updated the class name and its references.

**Review by Damovisa — APPROVED** [review]
Looks good, thanks!

**Pull request merged**